code/_onclick/hud/screentip.dm

```diff
@@ -6,6 +6,7 @@
 	maptext_height = 480
 	maptext_width = 480
 	maptext = ""
+	layer = OBJ_LAYER + 1 //Added to make screentips appear above action buttons
```

Comment:
It'd be better to add a unique layer define. It helps people working with layers in future know what's placed where

Reply:
The layer above OBJ_LAYER is mob layer. Would it be better to edit the line to just set it directly to MOB_LAYER?

Reply:
Thinking on it, you should give `/atom/movable/screen` a new, default layer define, then add another new layer for screentips

Reply:
Can you provide some tips for where I should put these new defines? I shouldn't just #Define SCREENTIP_LAYER in the screentip file right?

Reply:
stick em in `__DEFINES/layers.dm`, down near `HUD_PLANE`

Reply:
Oh of course, duh. Folder for defines probably has layer defines in it. 🤦 sorry

Reply:
sall good brother, our file structure was designed by satan

Reply:
Is there a way to update this pull request or do I have to close this one and make a new one?

Reply:
there is. You make a new git commit, and then "push" it from your local repo to github 
I'm not sure what git client you're using, so I can't tell you exactly the steps to do.

can ask me on discord if you'd like, or anyone in #coding-general

Reply:
I feel like because 3 is just the default layer that it wouldn't make much sense to add a #Define for just /atom/movable/screen, it's already on the HUD plane and has a layer of 3 by default. I have just added the define for the screentip layer though, I'll commit that and push it here.